miden-lib/asm/miden/kernels/tx/memory.masm

```diff
@@ -1013,13 +1027,13 @@ export.get_created_note_asset_data_ptr
     push.CREATED_NOTE_ASSETS_OFFSET add
 end
 
-#! Sets the created note vault hash
+#! Sets the created note assets hash.
 #!
-#! Stack: [created_note_data_ptr, V]
+#! Stack: [created_note_data_ptr, AH]
 #! Output: []
 #!
 #! - created_note_data_ptr is the memory address at which the created note data begins.
-#! - V is the vault hash of the created note.
-export.set_created_note_vault_hash
-    push.CREATED_NOTE_VAULT_HASH_OFFSET add mem_storew
+#! - AH is the assets hash of the created note.
```

Comment:
Is an Assets hash the same thing as a vault hash, considering that the vault is what stores assets? (Is it simply a naming change?)

Reply:
Correct. Old terminology is "vault hash" (or "vault root") the new terminology is "asset hash" - but they mean the same thing. We'll need to do a more comprehensive update to rename everything, but I'm leaving this for a future PR.